src/i18n.tsx

```diff
@@ -1,34 +1,77 @@
 import React from 'react'
 import { i18n } from '@lingui/core'
 import { I18nProvider } from '@lingui/react'
-import { detect, fromUrl } from '@lingui/detect-locale'
-import { ReactNode, useEffect } from 'react'
-
-export const locales = ['en', 'pseudo-en', 'de', 'es-AR', 'es-US', 'it-IT', 'iw', 'ro', 'ru', 'vi', 'zh-CN', 'zh-TW']
-export const defaultLocale = 'en'
-
-const getDetectedLocale = () => {
-  const detected =
-    detect(
-      fromUrl('lang'), // helps local development
-      defaultLocale
-    ) ?? defaultLocale
-  return locales.includes(detected) ? detected : defaultLocale
+import { ReactNode } from 'react'
+import { useDispatch } from 'react-redux'
+import useParsedQueryString from 'hooks/useParsedQueryString'
+import { useEffect } from '@storybook/addons'
+import { AppDispatch } from 'state'
+import { useLocale } from 'state/user/hooks'
+
+type IE11NavigatorLanguage = {
```

Comment:
big doubt we have users on ie11 but you can optionally also extend the navigator type definition in a declaration file

Reply:
i checked yesterday we had 13

Reply:
Seems like IE11 actually does support navigator.language:

"navigator.language is the same as navigator.userLanguage for IE."

https://zzz.buzz/2016/01/13/detect-browser-language-in-javascript/#navigatoruserlanguage